Assets/Prefabs/Player/Player.prefab

```diff
@@ -1,4958 +1,5 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
---- !u!1 &1003433082
```

Comment:
Player prefab is volatile, what change is this? What has been deleted from the Player prefab?

Reply:
I'm also in the middle of some significant changes to the player prefab so don't push this file. Check with me in person to sort the data out

Reply:
Okay lemme discuss with you on this